cstrml/market/src/lib.rs

```diff
@@ -308,10 +309,20 @@ decl_module! {
             <Merchants<T>>::mutate(&who, |maybe_minfo| {
                 if let Some(minfo) = maybe_minfo {
                     // Update merchant
+                    debug::info!(
```

Comment:
I don't think we should print out this `merchant` messages, because this already been included in `extrinsics` or `events` and this has no use for the network nodes. Actually we only print out the common info like consensus(election results), networking(syncing, connecting), otherwise there will be so many noisy info for the node

Reply:
Regarding to the happy path, I didn't add too much info. I can remove some of them as well like those in `update_ledger` and `batch_transfer`. Most of them are added for abnormal path.